plugins/modules/net_tools/nmcli.py

```diff
@@ -1760,6 +1760,7 @@ def main():
         ),
         supports_check_mode=True,
     )
+    module.run_command_environ_update = dict(LANG='C', LC_ALL='C', LC_MESSAGES='C', LC_CTYPE='C')
```

Comment:
@felixfontein you mentioned in the issue that lot of modules do this. What about setting the locale directly in the `module` so that everybody will benefit from it? Or are there any side-effects that could cause bigger issues?

Reply:
That would be nice, though it might break some modules which depend on these vars not to be set. I guess that's the main reason why every module has to do it by itself.